app/src/main/java/de/westnordost/streetcomplete/quests/note_discussion/NoteDiscussionForm.kt

```diff
@@ -140,22 +141,20 @@ class NoteDiscussionForm : AbstractQuestAnswerFragment<NoteAnswer>() {
             val userName = if (comment.user != null) comment.user.displayName else getString(R.string.quest_noteDiscussion_anonymous)
 
             val commentActionResourceId = comment.action.actionResourceId
-            if (commentActionResourceId != 0) {
-                itemView.commentStatusText.visibility = View.VISIBLE
+            val isStatusGone = commentActionResourceId == 0
```

Comment:
actually, if `commentActionResourceId == 0`, it means that it is a normal note comment without any additional action like "closed note", "reopened note" or "hidden note". I suggest renaming it to !`hasNoteAction`